pkg/cli/login/error_translation.go

```diff
@@ -22,8 +27,14 @@ You can also run this command again providing the path to a config file directly
 `
 	tlsOversizedRecordMsg = `Unable to connect to %[2]s using TLS: %[1]s.
 Ensure the specified server supports HTTPS.`
+	invalidServerURLMsg = `Seems you passed an HTML page (console?) instead of server URL.
+Verify provided address and try again.`
 )
 
+func (e *errInvalidServerURL) Error() string {
```

Comment:
Nit, let's keep it next to the type definition next time.

Reply:
done, had to address mo's comments so fixed : )